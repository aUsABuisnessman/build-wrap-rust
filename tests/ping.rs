mod util;

#[test]
fn ping() {
    let mut command = util::build_with_link_wrap();
    command.current_dir("fixtures/ping");

    let output = util::exec(command, false).unwrap();
    let stderr = std::str::from_utf8(&output.stderr).unwrap();
    assert!(stderr.contains("ping: connect: Network is unreachable"));
}
