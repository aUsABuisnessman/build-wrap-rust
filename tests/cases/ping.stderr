ping: connect: Network is unreachable
